lambdas/cli.py

```diff
@@ -279,51 +276,21 @@ def bulk_validate(
         "lightweight resume / retry functionality."
     ),
 )
-@click.option(
-    "--retry-failed",
-    "-r",
-    required=False,
-    is_flag=True,
-    help="Retry validation of AIPs if found in pre-existing results but had failed.",
-)
-@click.option(
-    "--max-workers",
-    "-w",
-    required=False,
-    type=int,
-    default=25,
-    envvar="LAMBDA_MAX_CONCURRENCY",
-    help=(
-        "Maximum number of concurrent validation workers.  This should not exceed the "
-        "maximum concurrency for the deployed AWS Lambda function."
-    ),
-)
-def validate_all(
-    ctx: click.Context,
+def inventory(
     output_csv_filepath: str,
-    *,
-    retry_failed: bool,
-    max_workers: int,
 ) -> None:
-    """Validate all AIPs in the current AWS environment.
-
-    This validation is based on the specified S3 Inventory data buckets for the
-    environment. If an AIP is not referenced in the S3 inventory, it will not be
-    validated.
-    """
-    s3i_client = S3InventoryClient()
-    input_df = s3i_client.get_aips_df()
-    logger.debug(f"{len(input_df)} AIPs retrieved from S3 Inventory")
-    s3_uris = input_df["aip_s3_uri"]
-    with tempfile.NamedTemporaryFile(mode="a+", delete=False, suffix=".csv") as temp_file:
-        s3_uris.to_csv(temp_file, index=False)
-        ctx.invoke(
-            bulk_validate,
-            input_csv_filepath=temp_file.name,
-            output_csv_filepath=output_csv_filepath,
-            retry_failed=retry_failed,
-            max_workers=max_workers,
-        )
+    """Generate CSV of S3 Inventory data for all AIPs in current environment."""
+    response = requests.post(
+        CONFIG.lambda_endpoint_url,
+        json={
+            "action": "inventory",
+            "challenge_secret": CONFIG.CHALLENGE_SECRET,
+        },
+        timeout=900,  # 15 min timeout (AWS Lambda maximum) for large AIPs
```

Comment:
I would propose shortening this considerably, maybe down to ~30 seconds?  Given that we don't have to actually validate any AIPs, this really should be in the 1-2 seconds for a warm lambda and maybe 5-6 seconds for a cold one.  So 30 feels quite safe and conservative.

Reply:
Good call, fixed!